apps/content_filters/filter_condition/filter_condition_parameters.py

```diff
@@ -137,6 +138,12 @@ def get(self, req, lookup):
                   {'field': 'anpa_take_key',
                    'operators': ['in', 'nin', 'eq', 'ne', 'like', 'notlike', 'startswith', 'endswith']
                    },
+                  {'field': 'agendas',
```

Comment:
actually you could add this field only if agenda is enabled, or even better create some api which planning could use to register a field..

Reply:
but given that it's the first external field, I would probably just make it conditional and comment on adding api if needed next time..

Reply:
Is there a way to check if agendas or enabled or not? If not I can simply check the length of `agenda_values` that is coming from here `agenda_values = self._get_planning_agendas()`. Would it be fine?

Reply:
I think it's this `if 'planning' in app.config.get('INSTALLED_APPS', []):` you do elsewhere